Makefile.toml

```
@@ -47,6 +47,7 @@ args = ["clippy", "--all-features", "--", "--deny", "warnings", "--deny", "clipp
 
 [tasks.fmt]
 description = "Format with rustfmt"
+install_crate = { rustup_component_name = "rustfmt", binary = "rustfmt", test_arg = "--help" }
```

```suggestion
install_crate = { rustup_component_name = "rustfmt", binary = "rustfmt", test_arg = "-V" }
```
It's a best practice to call the simplest and the most general command. In the most cases it's version getter.
Thanks!